deepchecks/core/suite.py

```diff
@@ -16,10 +16,12 @@
 import warnings
 from collections import OrderedDict
 from typing import Any, Dict, List, Optional, Sequence, Set, Tuple, Type, Union, cast
+import pathlib
 
 import jsonpickle
 from ipywidgets import Widget
 from typing_extensions import Self, TypedDict
+from bs4 import BeautifulSoup
```

Comment:
we might need to add it to the requirements

Reply:
so we need to move it from dev to requirements

Reply:
[done](https://github.com/deepchecks/deepchecks/pull/2147/commits/7fa61a51f519f97d2769f791262f5e17e234c769)

I kept it in dev regardless, as it seems someone else is using it for vision.

```sh
$ rg bs4 deepchecks
```
```
deepchecks/core/suite.py
24:from bs4 import BeautifulSoup

deepchecks/vision/datasets/detection/mask.py
28:from bs4 import BeautifulSoup
```